lib/chef/provider/chef_organization.rb

```diff
@@ -52,7 +52,9 @@ def whyrun_supported?
     new_resource.members.each do |user|
       if !existing_members.include?(user)
         converge_by "Add #{user} to organization #{new_resource.name}" do
-          rest.post("#{rest.root_url}/organizations/#{new_resource.name}/users/#{user}", {})
```

Comment:
OK!  I'll give her a shot as soon as I get some time.  Moving this to 1.0 so I don't forget.  We will get this in soon, in one way or the other :)  Thanks much for the issue find and fix.